src/components/AlphabetList/index.tsx

```diff
@@ -14,6 +14,7 @@ export const AlphabetList: React.FC<AlphabetListProps> = (props) => {
     data,
     index = DEFAULT_CHAR_INDEX,
     style,
+    indexContainerStyle,
```

Comment:
We need to add `indexContainerStyle` to the `AlphabetListProps` interface. 

You can find this file at `src/components/AlphabetList/types.ts`, and I reckon we can just add 

```ts
indexContainerStyle?: ViewStyle,
```